web/src/helpers/image.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable */
```

Comment:
I would put this in the solar component folder so we keep the solar component self-contained?

Reply:
I can do it if you want, but beware that `WindLayer` also has an external dependency `windy.js` and e.g. `AreaGraph` imports various methods from `helpers/*.js` :slightly_smiling_face: 

My idea here was that we'd be able to get rid of this file completely once we use e.g. WebGL blur filters as a part https://github.com/tmrowco/electricitymap-contrib/issues/923 and in the meantime I wanted to keep a clean separation between our internal `SolarLayer` React component and the temporary "external" blur helper which seems to be an adaptation of https://github.com/flozz/StackBlur and was never properly integrated in our code.

If you expect we'll be depending on that helper for a while, perhaps I can actually remove it in favor of using https://github.com/flozz/StackBlur directly, that would also somewhat simplify the code in `solarlayer.js`?

Reply:
hmm let's keep it here for now. We're getting a bit pressed for time so I don't want to focus too much on these aspects which can be changed later as well :)